webextensions/api/sessions.json

```
@@ -270,7 +270,7 @@
             "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/sessions/restore",
             "support": {
               "chrome": {
-                "version_added": "≤58"
+                "version_added": "30"
```

:x: Like parent feature.

```suggestion
                "version_added": "37"
```